esphome/components/nextion/display.py

```diff
@@ -35,7 +37,7 @@
     display.BASIC_DISPLAY_SCHEMA.extend(
         {
             cv.GenerateID(): cv.declare_id(Nextion),
-            cv.Optional(CONF_TFT_URL): cv.All(cv.string, cv.only_with_arduino),
+            cv.Optional(CONF_TFT_URL): cv.All(cv.string),
```

Comment:
Maybe change this to validate as a url?
```suggestion
            cv.Optional(CONF_TFT_URL): cv.url,
```

Reply:
I will have to run some tests as this is changing from the original code.
Will try that tomorrow.

Reply:
Thanks!